src/Symfony/Component/Notifier/Bridge/LineNotify/composer.json

```
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/line-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Provides LINE Notify integration for Symfony Notifier.",
+    "keywords": ["LINE", "notifier", "chat", "sms"],
```

```suggestion
    "keywords": ["line", "notifier", "chat", "sms"],
```
Is it about chat or sms?